diff --git a/fritzing-app/tools/release_fritzing.bat b/fritzing-app/tools/release_fritzing.bat
index f8e758a8..30527ea1 100644
--- a/fritzing-app/tools/release_fritzing.bat
+++ b/fritzing-app/tools/release_fritzing.bat
@@ -38,34 +38,15 @@ echo add the path for your git installation if it's not already there
 set PATH=%PATH%;"C:\Program Files (x86)\Git\bin";
 
 echo set the path to the qt sdk bin folder
+set QTBIN=%RUNNER_WORKSPACE%\Qt\5.15.2\mingw81_64\bin
 IF %2==64 (
-	IF %3==2012 (
-	set QTBIN=C:\Qt\5.6\msvc2012_64\bin
-    ) ELSE IF %3==2013 (
-	set QTBIN=C:\Qt\5.6\msvc2013_64\bin
-    ) ELSE IF %3==2015 (
-	set QTBIN=C:\Qt\5.6\msvc2015_64\bin
-    ) ELSE IF %3==2017 (
-        set QTBIN=C:\Qt\5.12.7\msvc2017_64\bin
-    )
-	set arch=""QMAKE_TARGET.arch=x86_64""
+	set arch="QMAKE_TARGET.arch=x86_64"
+) ELSE IF %2==32 (
+	set arch=.
 ) ELSE (
-	IF %2==32 (
-		IF %3==2012 (
-		set QTBIN=C:\Qt\5.6\msvc2012\bin
-	    ) ELSE IF %3==2013 (
-		set QTBIN=C:\Qt\5.6\msvc2013\bin
-	    ) ELSE IF %3==2015 (
-		set QTBIN=C:\Qt\5.6\msvc2015\bin
-	    ) ELSE IF %3==2017 (
-		set QTBIN=C:\Qt\5.12.3\msvc2017\bin
-	    )
-		set arch=.
-	) ELSE (
-		echo second parameter--target architecture--should be either "32" for a 32-bit build or "64" for a 64-bit build
-		dir %RUNNER_WORKSPACE%\Qt
-		EXIT /B 1
-	)
+	echo second parameter--target architecture--should be either "32" for a 32-bit build or "64" for a 64-bit build
+	dir C:\Qt
+	EXIT /B 1
 )
 
 set QMAKE=%QTBIN%\qmake.exe
